insights/parsers/ironic_conf.py

```diff
@@ -0,0 +1,32 @@
+"""
+IronicConf - file ``/etc/ironic/ironic.conf``
+================================================
```

Comment:
When you test your docs, you'll have an error here if the underline is not the same length as the line above it.